internal/commands/relationship_test.go

```diff
@@ -549,3 +559,136 @@ func (m *mockClient) WriteRelationships(_ context.Context, in *v1.WriteRelations
 	require.True(m.t, proto.Equal(expectedWrite, in))
 	return &v1.WriteRelationshipsResponse{WrittenAt: &v1.ZedToken{Token: "test"}}, nil
 }
+
+func TestBulkDeleteForcing(t *testing.T) {
```

Comment:
Can we add a test with, say, a couple hundred rels?

Reply:
@josephschorr done in https://github.com/authzed/zed/pull/365/commits/5f573aa5af5fd8474263516f04e41441c04836fd, PTAL 🙏🏻